sherpa/optmethods/__init__.py

```diff
@@ -116,751 +116,9 @@
    const1d.c0   thawed            4 -3.40282e+38  3.40282e+38
 
 """
+from sherpa.optmethods import buildin
+from sherpa.optmethods import optscipy
+from sherpa.optmethods.buildin import *
+from sherpa.optmethods.optscipy import *
```

Comment:
We could change this to (untested, so I'm just assuming it will work)

```python
from . import buildin, optscipy
from .buildin import *
from .optscipy import *
```

Is it worth it? Will it break things?

Reply:
Actually, the `optscipy` lines should only be added in the next commit, not this one.

Reply:
Either relative of absolute imports work. I don't feel strongly either way. Yes, the optscipy line should have been added in the next commit - it happened when the reorganized the commits as requested earlier in the review. will try to fix. (Though nobody will check out intermediate commits...)

Reply:
Done while rewriting the commit history to fix the commit at which optscipy is imported.